spec/project_spec.rb

```diff
@@ -146,9 +142,7 @@
             entry.match(/^\*\s*\[/)
           end
 
-          entries_including_issue_link.each do |entry|
-            expect(entry).to include('): ')
-          end
+          expect(entries_including_issue_link).to all(be_include('): '))
```

Comment:
`be_include` doesn’t sound right. But I’m not sure what would be another way to fix it.

Reply:
`all(include('): '))` might work. I'm pretty sure I've used `all(include(...))` before.